packages/fluentui/react-northstar/test/specs/components/Popup/Popup-test.tsx

```diff
@@ -253,7 +253,7 @@ describe('Popup', () => {
       // reset body, because then "firstElementChild" was not properly getting right element
       document.body.innerHTML = '';
       mountWithProvider(<Popup trigger={<button />} content="Content" open />);
-      const contentElement = document.body.firstElementChild;
+      const contentElement = document.body.firstElementChild.firstElementChild;
```

Comment:
- this test was mounted with a default context where `target` (i.e. `document` is present)
- this was mounted without`PortalBoxContext` that contained an element to `mount`

That's why before a portal was mounted directly to `document.body` (see logic in `PortalInner`).